docs/README.md

```diff
@@ -122,7 +122,7 @@ The navigation toolbar can be turned off in the [Settings panel](#settings-panel
 
 There is a compass on the display's lower-right corner. Its red pointer points to north. Touching the compass will rotate the map so the top points toward north.
 
-The [compass tool] is part of ArcGIS Runtime Toolkit for Qt.
+The [NorthArrow compass control] is part of ArcGIS Runtime Toolkit for Qt.
```

Comment:
Perhaps we link users to the Toolkit from this line?

Reply:
I've left the specific reference to the NorthArrow component while still supplementing this line with a link to the Toolkit repo. Hopefully this is what you meant. I do think both links serve a purpose here.